pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -203,6 +244,10 @@ spec:
                       but this parameter is not specified, the default value is 2.
                     format: int64
                     type: integer
+                    x-kubernetes-validations:
```

Comment:
It it possible to disable this field? If so, then 0 should be disabled.

Reply:
What do you mean by disable? 0 will not be considered as null value